packages/web-components/fast-foundation/src/radio-group/radio-group.spec.ts

```diff
@@ -266,6 +266,42 @@ describe("Radio Group", () => {
         await disconnect();
     });
 
+    it("should mark the last radio defaulted to checked as checked, the rest should not be checked", async () => {
+        const { element, connect, disconnect } = await fixture(html<FASTRadioGroup>`
```

Comment:
You don't need the type on the html function in this case. That would only be needed if you had a special view model that you wanted to test *binding* the radio group to.

Reply:
I removed it for my specs, as I don't need to cast them specifically to FASTRadioGroup.